src/main/resources/api/service_openapi.yaml

```diff
@@ -23,6 +23,29 @@ paths:
         500:
           description: Service is broken
           $ref: '#/components/responses/StatusResponse'
+  '/api/v1/folders':
+    post:
+      description: |
+        Asynchronous endpoint to create a new empty folder. Result of this async
+        job will be a CreatedFolder object.
+      operationId: createFolder
+      tags:
+      - folder
+      requestBody:
+        description: |
+          Name of the new folder to be created, as well as the ID of the
+          containing folder (absent if this is a top-level folder).
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateFolderBody'
+      responses:
+        202:
+          description: Accepted
+          $ref: '#/components/responses/JobModelResponse'
+        500:
+          description: Creation request error
+          $ref: '#/components/responses/ErrorResponse'
```

Comment:
see previous comment: probably need to add a 4xx code here if the user supplied an invalid folder name, spend profile id,  etc.

Reply:
Done